velox/exec/tests/utils/PlanBuilder.cpp

```diff
@@ -303,23 +303,30 @@ PlanBuilder& PlanBuilder::tableWrite(
     const std::shared_ptr<core::AggregationNode>& aggregationNode,
     const std::shared_ptr<core::InsertTableHandle>& insertHandle,
     bool hasPartitioningScheme,
-    CommitStrategy commitStrategy) {
+    CommitStrategy commitStrategy,
+    RowTypePtr outputType) {
   planNode_ = std::make_shared<core::TableWriteNode>(
       nextPlanNodeId(),
       inputColumns,
       tableColumnNames,
       aggregationNode,
       insertHandle,
       hasPartitioningScheme,
-      TableWriteTraits::outputType(),
+      outputType == nullptr ? TableWriteTraits::outputType() : outputType,
```

Comment:
Can we extend outputType() take an optional aggregationNode to generate the output type?